test/upgrade/upgrade_test.go

```diff
@@ -54,6 +54,7 @@ func TestServerlessUpgrade(t *testing.T) {
 				},
 				kafkaupgrade.ChannelContinualTests(continual.ChannelTestOptions{}),
 				kafkaupgrade.SourceContinualTests(continual.SourceTestOptions{}),
+				kafkabrokerupgrade.BrokerContinualTests(),
```

Comment:
should we delete this too and maybe enable the broker from the start of the test?
```golang
					// Remove this when upgrading from Serverless 1.20 to 1.21.
					if err := installation.EnableKafkaBroker(ctx); err != nil {
						c.T.Error("Failed to enable Kafka Broker on KnativeKafka resource:", err)
					}
```

Reply:
Right, I was too quick. Thanks